gengokit/middlewares/templates/endpoints.go

```diff
@@ -4,21 +4,21 @@ const EndpointsBase = `
 package middlewares
 
 import (
-	"github.com/go-kit/kit/endpoint"
 	svc "{{.ImportPath -}} /generated"
 )
 
-// WrapEndpoint will be called individually for all endpoints defined in
-// the service. Implement this with the middlewares you want applied to
-// every endpoint.
-func WrapEndpoint(in endpoint.Endpoint) endpoint.Endpoint {
-	return in
-}
-
 // WrapEndpoints takes the service's entire collection of endpoints. This
 // function can be used to apply middlewares selectively to some endpoints,
 // but not others, like protecting some endpoints with authentication.
 func WrapEndpoints(in svc.Endpoints) svc.Endpoints {
+
+	// Apply a middleware selectively
+	// in.Echo = endpoint.Middleware(in.Echo)
```

Comment:
I understand the intent, but again, `endpoint.Endpoint` is a type, not a callable function. What is literally shown here is a type conversion, not a function call. This is not semantically legal Go, but it is syntactically legal Go, so the intent gets muddied in this ambiguity. This is why I think a dummy, example middleware name would be clearer. 

The out-of-placeness of `in.Echo`--a service that you are well familiar with as Truss' author, but which may not mean anything to a user--compared to an illustrative example from the actual service being rendered, remains, as mentioned in the second paragraph of the original comment.